packages/client/hmi-client/src/components/model/petrinet/model-configurations/tera-stratified-matrix.vue

```diff
@@ -301,6 +324,16 @@ onUnmounted(() => {
 	window.clearInterval(timerId);
 });
 
+// This just prepares the filters when the matrix is assigned for the first time
+watch(
+	() => matrix.value,
```

Comment:
No need to change it, but you can do just like this to watch a computed or ref.

```suggestion
	matrix,
```